### ISSUE TITLE
Traincrafts own switch motors and signals

### ISSUE THREAD

**Issue by 31trainman** [EternalBlueFlame/Traincraft-5#105]
I say we should make our own switch motors and signals etc. I think reliance upon a third party mod for stuff is kinda silly when this is a train mod that should have it in it.

**Comment by EternalBlueFlame** [comment]
you can use **any** redstone based devices, there are a plethora of mods for redstone switches and whatnot, buildcraft gates and wires as one example.

**Comment by Larky2k** [comment]
Railcraft Switches don't work with Traincraft 3d rails using latest comits.

**Comment by NitroxydeX** [comment]
@Larky2k true. because they don't emit a redstone signal. :) we are only accepting things from redstone signals. Signals from Railcraft arent working too because we dont extend base-game-rails.

**Comment by Larky2k** [comment]
just saw your post from a few hours ago. hopefully down the line they will work.

**Comment by EternalBlueFlame** [comment]
There are plans to rework the tracks eventually so this sort of thing would work, but I wouldn't hold my breath for something soon.

**Comment by Cronosus** [comment]
one useable switch (lever) from me
https://www.dropbox.com/sh/hybmhq3fvmkg3wu/AADC5TRvmRA1z96-w4IQ13EZa?dl=0
![myswitch](https://cloud.githubusercontent.com/assets/5349950/18443961/8a1a66fe-7917-11e6-96c5-d8018f8e2bc5.png)
